stm/asyncTaskQueue.ml

```diff
@@ -132,6 +132,7 @@ module Make(T : Task) () = struct
         | ( "-I" | "-nI" | "-include" | "-top" | "-topfile" | "-coqlib" | "-exclude-dir"
           | "-color" | "-init-file"
           | "-profile-ltac-cutoff" | "-main-channel" | "-control-channel"
+          | "-native-output-dir" | "-w"
```

Comment:
What is native-output-dir?

Reply:
it control where coqc put the cmxs, the default is `.coq-native`

Reply:
Done in #15991